Dockerfile

```diff
@@ -49,7 +49,7 @@ ENV FLASK_APP=/code/reana_server/app.py
 EXPOSE 5000
 
 CMD flask db init && \
-    flask users create info@reana.io &&\
+    flask users create_default info@reana.io &&\
```

Comment:
Note this will also need to be changed in `reana-cluster` if the CMD for the container is specified there.

Reply:
Present in `reana-cluster` master see https://github.com/reanahub/reana-cluster/pull/99.

Reply:
It's actually already merged: https://github.com/reanahub/reana-cluster/pull/99/files